lib/asciidoctor/ietf/validate.rb

```diff
@@ -34,59 +34,8 @@ def validate(doc)
                         File.join(File.dirname(__FILE__), "ietf.rng"))
       end
 
-      def wgcache_name
-        "#{Dir.home}/.metanorma-ietf-workgroup-cache.json"
-      end
-
-      def open_wg_cache(node)
-        node.attr("flush-caches") == "true" and
```

Comment:
@ronaldtse @opoudjis I wanna to make your attention that "flush-caches" logic removed, please let me know if it's important for us.

Our data will be updated once per day. Also, there is a button to update right now https://github.com/metanorma/metanorma-ietf-data/actions/workflows/check_update.yml

So as far as I understand we don't need this logic anymore, but I may be wrong

Reply:
I believe this functionality should not be removed. Our data may be being updated a lot more frequently than IETF's, but it is still a legitimate requirement to operate offline and to cache workgroups.

@ronaldtse?

Reply:
@opoudjis misunderstanding? There is a metanorma-ietf-data gem that is automatically published when there are updates to the lists, that provides offline data.